Newer rust compiler is smarter (then our MSRV) apparently:
```
error[E0716]: temporary value dropped while borrowed
   --> components/script/dom/readablestream.rs:219:39
    |
219 |         let realm = InRealm::Already(&cx.into());
    |                                       ^^^^^^^^^ - temporary value is freed at the end of this statement
    |                                       |
    |                                       creates a temporary value which is freed while still in use
...
274 |         self.check_and_propagate_errors_forward(cx, &global, realm, can_gc);
    |                                                              ----- borrow later used here
    |
help: consider using a `let` binding to create a longer lived value
    |
219 ~         let binding = cx.into();
220 ~         let realm = InRealm::Already(&binding);
    |
```

--- a/components/script/dom/readablestream.rs
+++ b/components/script/dom/readablestream.rs
@@ -216,7 +216,8 @@ impl Callback for PipeTo {
     #[expect(unsafe_code)]
     fn callback(&self, cx: &mut CurrentRealm, result: SafeHandleValue) {
         let can_gc = CanGc::from_cx(cx);
-        let realm = InRealm::Already(&cx.into());
+        let in_realm_proof = cx.into();
+        let realm = InRealm::Already(&in_realm_proof);
         let cx = cx.into();
         let global = self.reader.global();
 

